### PR TITLE
8.10.3 Release notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/3991.

Previews:
- [Index page](https://security-docs_4011.docs-preview.app.elstc.co/guide/en/security/master/release-notes.html)
- [Release notes](https://security-docs_4011.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.10.1.html#release-notes-8.10.3)

### CODE REVIEW

**Comment by nastasha-solomon** [comment]
@elasticmachine run elasticsearch-ci/docs

**Review by PhilippeOberti — APPROVED** [review]
All good for the flyout changes, thanks @nastasha-solomon !

File: docs/release-notes/8.10.asciidoc [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+[discrete]
+[[enhancements-8.10.3]]
+==== Enhancements
+* Updates the MITRE ATT&CK® framework to `v13.1` ({pull}166536[#166536]).
+
+[discrete]
+[[bug-fixes-8.10.3]]
+==== Bug fixes
+* Increases the line height of the session view preview in the alert details flyout ({pull}166932[#166932]).
+* Adds tooltips to the **Correlations** table in the alert details flyout ({pull}166913[#166913]).
```
Comment (nastasha-solomon) [inline docs/release-notes/8.10.asciidoc:17]:
```suggestion
* Adds tooltips to the Correlations table in the alert details flyout ({pull}166913[#166913]).
```

File: docs/release-notes/8.10.asciidoc [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
 == 8.10
 
+[discrete]
+[[release-notes-8.10.3]]
+=== 8.10.3
+
+[discrete]
+[[enhancements-8.10.3]]
+==== Enhancements
+* Updates the MITRE ATT&CK® framework to `v13.1` ({pull}166536[#166536]).
```
Comment (nastasha-solomon) [inline docs/release-notes/8.10.asciidoc:11]: Addresses https://github.com/elastic/security-docs/issues/4013.

**Review by benironside — APPROVED** [review]
LGTM, thanks!

**Review by dplumlee — APPROVED** [review]
mitre update looks good!

File: docs/release-notes/8.10.asciidoc [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
+
+[discrete]
+[[bug-fixes-8.10.3]]
+==== Bug fixes
+* Increases the line height of the session view preview in the alert details flyout ({pull}166932[#166932]).
+* Adds tooltips to the Correlations table in the alert details flyout ({pull}166913[#166913]).
+* Fixes a bug that prevented the prevalence query from considering fields with multiple values ({pull}166891[#166891]).
+* Fixes an alert details flyout bug that affected the way prevalence data was fetched ({pull}166694[#166694]).
+* Fixes issues with the **Top alerts by** inline action within Timeline ({pull}165109[#165109]).
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/release-notes/8.10.asciidoc:20]: Thanks for this, @machadoum! I'm updating the 8.10.3 summary and adding an 8.10.4 entry [here](https://github.com/elastic/security-docs/pull/4024/files).

**Pull request merged**